app/helpers/notify_view_helper.rb

```diff
@@ -4,7 +4,7 @@ def notify_link(url, text = url)
   end
 
   def unsubscribe_link(token)
-    link_text = t('subscriptions.email.unsubscribe_link_text')
+    link_text = I18n.t('subscriptions.email.unsubscribe_link_text')
```

Comment:
Why is this change necessary?

Reply:
We made this change because one of the tests complained about it. 
Ran the test suite again (as of the time of writing this) with this change removed. 
Tests work fine. 
Must have been left over from brainstorming how to get the tests to pass.